src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/NoProvidersAlert.jsx

```diff
@@ -1,39 +1,29 @@
 import React from 'react';
-import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 
 export default function NoProvidersAlert({ sortMethod, typeOfCareName }) {
-  const headline = `We can’t find any ${typeOfCareName} providers close to you`;
   return (
-    <AlertBox
-      status="info"
-      headline={headline}
-      className="vads-u-margin-top--3"
-      content={
-        <>
-          <p>
-            We can’t find providers within 60 miles from your{' '}
-            {sortMethod ? 'current location' : 'home'}. To request this
-            appointment, you can: <br />
-          </p>
-          <ul>
-            <li>
-              Call your VA or community care facility.{' '}
-              <a
-                href="/find-locations"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Find your health facility’s phone number
-              </a>
-              , <strong>or</strong>
-            </li>
-            <li>
-              Continue your request without choosing a provider. We’ll contact
-              you about about a provider.
-            </li>
-          </ul>
-        </>
-      }
-    />
+    <div className="vads-u-margin-top--2 vaos-appts__block-label vads-u-background-color--primary-alt-lightest vads-u-padding--2p5">
+      <h3 className="vads-u-margin-top--0">
+        We can't find any {typeOfCareName} providers close to you
+      </h3>
+      <p>
+        We can’t find any providers within 60 miles from your{' '}
+        {sortMethod ? 'current location' : 'home'}. To request this appointment,
+        you can: <br />
+      </p>
+      <ul>
+        <li>
+          Call your VA or community care facility.{' '}
+          <a href="/find-locations" target="_blank" rel="noopener noreferrer">
```

Comment:
Needs to announce to a screen reader that it opens in a new window. In this case: `aria-label="Find your health facility’s phone number (Opens in new window)" ` should suffice

Reply:
Just adding a resource here for any further research https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/links/links-and-new-windows/